org/all-prs.js

```diff
@@ -60,10 +60,10 @@ const approvedVerbs = [
   'Release',
 ];
 
-const linkForCommit = commit => `[${commit.sha.slice(0, 6)}](${commit.url})`;
+const linkForCommit = commit => `[\`${commit.sha.slice(0, 6)}\`](${commit.html_url})`;
```

Comment:
Darn it. I was using https://developer.github.com/v3/repos/commits/ for reference about the commit object, because I figured Danger uses the GitHub API. Turns out the reference is here: https://danger.systems/js/reference.html#GitHubCommit, and Danger doesn't actually expose the html url..

Reply:
Ah, sorry for not looking into that myself either. Anyway I thought in the past the url pointed to the browsable one, but now I notice it hasn't for at least some time.

Let's revert this and I'll try to look into danger source/open an issue later.

Reply:
> Let's revert this

We haven't really made it worse though